### DISCUSSION TITLE
Github PR file filters work terribly with Unity projects

### DISCUSSION THREAD

**Discussion by KevinGelkingSviper** [community/community#41901] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Heya,

We're using Github as our repo host for a few projects (all Unity games) and are rather displeased with the file filtering options Github (the page itself) offers for pull requests.

The three main issues:
- we cannot specify a default filter in the project settings (pre-filter files for .cs, .glsl, .hlsl and .manifest)
- every reviewer has to manually un-tick every 'non-relevant' item to leave only the relevant one/s
- due to the large amount of irrelevant files the code-preview does not work for the files that matter, this means every reviewer also has to click  every single code file manually to make it load the preview

A configurable standard filter that can be set in the project settings is desperately needed and would avoid all three issues above.

See picture attached for a typical view of how it's like to work with Unity projects on Github PRs.
Is there a setting we missed where this can be achieved? Default filters are possible in, for instance, Bitbucket.
![image](https://user-images.githubusercontent.com/103119563/208080985-3aff2465-d07c-40f9-ba0a-ca4a9b330b7b.png)

**Comment by vgilabert** [comment]
I wonder why this post has fallen into oblivion, since I feel like it's bothering a lot of Unity developers. I wish to add my voice to yours and hope someone will see this, even if it's 2.5 years later.
